packages/icons/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@clr/icons",
+  "version": "3.1.0",
```

Comment:
When we publish will we need to update the version in all the `package.json` files now?

Reply:
Currently yes, I have already put a note on my list to make this easier somehow.

Reply:
No worries. That is not a hardship. But could you add it to the release process documentation?